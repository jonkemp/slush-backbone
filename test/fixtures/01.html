<html>
<head>
    <title></title>
</head>
<body>
<!-- build:js scripts/main.js -->
<script src="scripts/main.js"></script>
<!-- endbuild -->
</body>
</html>