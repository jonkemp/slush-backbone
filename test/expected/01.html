<html>
<head>
    <title></title>
</head>
<body>
<!-- build:js scripts/main.js -->
<script src="scripts/main.js"></script>
<script src="scripts/models/application.js"></script>
<!-- endbuild -->
</body>
</html>